docs/quickstarts/packet.md

```diff
@@ -45,99 +45,80 @@ The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provi
-refer the DNS configuration settings for Packet(Add link).
+Extract the binary and copy it to a place under your `$PATH`:
 
 ```console
-export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
-export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+tar zxvf lokoctl_0.1.0_linux_amd64.tar.gz
+sudo cp lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin
 ```
 
-### Step 4: Define cluster configuration
-
-To create a Lokomotive cluster, we need to define a configuration.
-
-A [production-ready configuration](../../examples/packet-production) is already provided for ease of
-use. Copy the example configuration to the working directory and modify accordingly.
-
-The provided configuration installs the Lokomotive cluster and the following components:
+### Step 2: Create a cluster configuration
 
-* [metrics-server](../configuration-reference/components/metrics-server.md)
-* [openebs-operator](../configuration-reference/components/openebs-operator.md)
-* [contour](../configuration-reference/components/contour.md)
-* [metallb](../configuration-reference/components/metallb.md)
-* [cert-manager](../configuration-reference/components/cert-manager.md)
-* [flatcar-linux-update-operator](../configuration-reference/components/flatcar-linux-upate-operator.md)
-* [openebs-storage-class](../configuration-reference/components/openebs-storage-class.md)
-* [prometheus-operator](../configuration-reference/components/prometheus-operator.md)
+Create a directory for the cluster-related files and navigate to it:
 
-You can configure the components as per your requirements.
+```console
+mkdir lokomotive-demo && cd lokomotive-demo
+```
 
-Create a variables file named `lokocfg.vars` in working directory to set values for variables
-defined in the configuration file.
+Create a file named `cluster.lokocfg` with the following contents:
 
-```console
-#lokocfg.vars
+```hcl
+cluster "packet" {
+  asset_dir        = "./assets"
+  cluster_name     = "lokomotive-demo"
 
-packet_project_id = "PACKET_PROJECT_ID"
-ssh_public_keys = ["ssh-rsa AAAAB3Nz...", "ssh-rsa AAAAB3Nz...", ...]
+  dns {
+    zone = "example.com"
+    provider {
+      route53 {
+        zone_id = "X27BF86A3BNP80"
+      }
+    }
+  }
 
-state_s3_bucket = "name-of-the-s3-bucket-to-store-the-cluster-state"
-lock_dynamodb_table = "name-of-the-dynamodb-table-for-state-locking"
+  facility = "ams1"
+  project_id = "89273817-4f44-4b41-9f0c-cb00bf538542"
 
-dns_zone = "dns-zone-name"
-route53_zone_id = "zone-id-of-the-dns-zone"
+  ssh_pubkeys       = ["ssh-rsa AAAA..."]
+  management_cidrs  = ["0.0.0.0/0"]
+  node_private_cidr = "10.0.0.0/8"
 
-management_cidrs = "public-ip-address-cidr-to-access-the-cluster"
-node_private_cidr = "private-subnet-assigned-by-packet-to-the-project"
+  controller_count = 1
 
-cert_manager_email = "email-address-used-for-cert-manager-component"
-grafana_admin_password = "password-for-grafana"
+  worker_pool "pool-1" {
+    count     = 2
+    node_type = "t1.small.x86"
```

Comment:
Commit message says:

> Include sample config in the guide with the absolute minimum set of
  knobs the user needs to worry about. Use defaults as much as
  possible.

But node_type is that by default in lokomotive-kubernetes assets, so no need to specify it. If you just want to show how to change the instance type (IMHO, it seems reasonable) seems okay to keep it. But maybe mention it in the commit message?

Like this, it seems like an unintended change :)

Reply:
I've included it because I would feel uneasy about not knowing the machine type `lokoctl` is going to create. What are you proposing?

Reply:
Sorry I wasn't clear. My suggestion, if you want to leave this, is to change the commit msg:
> But maybe mention it in the commit message?

Reply:
I've addressed this by mentioning that we use `t1.small.x86` for all nodes in the introduction and removing `node_type` from the sample config.

Reply:
:ok_hand: